proposals/003-installer/installer-guidelines.md

```diff
@@ -0,0 +1,79 @@
+# Installer Guidelines
+
+## 0. Objectives
+
+We want people to use Haskell, which means the path to installing Haskell for first-time users
+should be short and easy, and it should yield an environment that is easy to use with the supporting
+literature. In addition, building simple programmes with a fresh installation should be as quick
+and painless as possible.
+
+These guidelines will set out the criteria that the Haskell Foundation will use in evaluating
+installers. We do not expect any individual to meet all these criteria initially but we would like
+them to be met by a collection of installers as quickly as possible.
+
+## 1. Cross Platform Coverage
+
+As many platforms should be covered by the installers as possible. For now, the minimum platforms
+that should be covered should include 64-bit Windows, macOS and Linux running on
+Intel-x86-compatible processors. This set of minimal platforms will soon have to be extended to
+include ARM-architecture processors.
+
+## 2. Uniformity of Delivered Environment
+
+Each installer should deliver a Haskell development environment that includes the following.
+
+  * The current recommended GHC toolchain, as determined by the installer, should be installed out
+    of the box, ready to use by all of the installed build systems.
+
+  * Both of the principal build systems (`cabal-install` and `stack`) should be installed,
+    available and ready for use in the way their associated development communities would expect.
+    (If an installer interrogates the user on installation about which components to install it is
+    fine to make `cabal-install` or `stack` optional, provided are both installed when the default
+    options are take.)
+
+  * It should be possible to install and manage multiple toolchains that both `stack` and
+    `cabal-install` can use. This requires that both `stack` and `cabal-install` need to be
+    configurable to access a GHC toolchain that has been installed by the other and the installer
+    must configure both toolchains to work with the `starter` GHC toolchain.
```

Comment:
This doesn't make much sense, because you can tell both stack and cabal which GHC to pick.

I feel that something else here is meant though and I believe it has no place in this proposal. This is an optimization/configuration concern.